File: BackEnd/src/main/java/com/Checkerly/BackEnd/CheckerlyApplication.java
package com.Checkerly.BackEnd;

import org.springframework.boot.SpringApplication; 
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;
import org.springframework.mail.javamail.JavaMailSender;
import org.springframework.mail.javamail.JavaMailSenderImpl;
import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
import org.springframework.security.crypto.password.PasswordEncoder;

import java.util.Properties;

@SpringBootApplication
public class CheckerlyApplication {
    public static void main(String[] args) {
        SpringApplication.run(CheckerlyApplication.class, args);
    }
}

File: BackEnd/src/main/kotlin/com/Checkerly/BackEnd/CheckerlyApplication.kt
package com.Checkerly.BackEnd

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
object CheckerlyApplication {
    fun main(args: Array<String?>?) {
        SpringApplication.run(CheckerlyApplication::class.java, args)
    }
}
